### DISCUSSION TITLE
Updated to Node.js v18. wont dissapear

### DISCUSSION THREAD

**Discussion by lyhnet** [wg-easy/wg-easy#788] [Q&A] [answered]
After running 'docker compose pull' and restarting container i still get 'Updated to Node.js v18.' in the web-interface:
![image](https://github.com/wg-easy/wg-easy/assets/33951833/65107d3e-d0b5-461e-bd82-7b11679d747e)

How do i update?

Thanks

**Comment by eandersons** [comment, marked as answer]
What image are you using - `wg-easy/wg-easy` or `ghcr.io/wg-easy/wg-easy`?
The latter should contain the updated Node.js.

See also #681

**Reply by lyhnet** [reply]
Thanks, that helped.

**Reply by Fastidious** [reply]
I am using the later, and the error persists.

**Reply by Fastidious** [reply]
That is, I am doing:

```
docker stop wg-easy
docker rm wg-easy
docker pull ghcr.io/wg-easy/wg-easy
```

And running my `docker run -d --name=wg-easy ...`, and the notice still shows.

**Reply by Fastidious** [reply]
Never mind, fix it. At the end of the `docker run -f --name=wg-easy ...`  I still had `wg-easy/wg-easy` instead of `ghcr.io/wg-easy/wg-easy`. All good now.
